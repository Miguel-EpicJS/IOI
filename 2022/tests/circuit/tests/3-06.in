385045b28fbc1903eca997d513040235e0229e01
1000 1000 5
-1 0 0 0 2 2 2 6 6 5 2 9 6 12 10 8 15 15 14 8 2 7 12 6 5 14 25 0 10 20 20 6 16 3 33 31 15 11 31 10 0 36 27 26 43 27 38 31 42 10 2 24 50 8 2 24 55 55 44 6 51 2 38 61 24 42 22 57 31 57 67 38 65 24 15 24 52 6 23 56 6 51 55 71 18 84 80 54 76 71 75 64 78 62 55 45 16 75 73 63 56 100 51 23 91 86 12 103 69 61 77 55 55 87 29 86 90 82 96 36 81 54 11 105 112 121 88 55 35 28 50 129 62 124 46 116 42 113 69 138 15 119 16 121 86 134 106 132 98 80 36 23 149 76 2 116 53 55 50 95 143 139 82 117 121 149 160 23 113 42 168 157 154 117 159 121 173 143 175 132 149 80 94 131 110 121 163 164 62 87 172 190 167 133 156 140 116 149 124 27 181 200 123 90 164 180 175 86 188 175 68 143 195 128 69 173 82 26 199 207 9 167 142 132 57 209 173 172 164 121 123 143 166 165 113 136 211 175 214 105 23 171 239 151 212 218 209 172 90 48 246 115 187 206 180 157 237 207 117 181 175 64 255 177 213 122 239 165 253 160 210 247 179 172 197 124 29 196 172 247 206 114 241 172 273 121 181 271 238 259 241 285 255 278 293 236 293 231 262 239 137 121 167 298 283 278 149 297 231 202 229 87 197 25 248 111 243 269 93 181 184 110 217 257 291 182 118 230 315 149 182 270 201 257 162 224 284 312 203 313 322 324 259 259 121 158 305 327 182 231 269 317 349 338 207 334 185 197 231 354 137 354 190 180 294 155 291 307 158 214 363 207 273 259 259 239 366 326 282 105 237 306 374 355 292 381 379 342 274 253 323 166 375 312 186 342 174 370 240 343 326 57 300 314 357 82 239 315 197 408 147 146 258 363 379 11 130 247 234 175 409 408 279 134 362 274 103 274 346 333 428 77 408 397 188 374 128 419 433 196 308 167 55 337 410 213 276 160 172 387 432 327 431 444 414 373 432 51 418 450 381 184 459 379 369 175 412 374 395 439 453 387 87 379 284 371 444 188 331 440 231 474 407 436 482 480 349 436 473 379 408 350 407 193 383 475 402 274 432 409 409 401 278 418 385 482 180 457 315 112 215 493 426 450 247 337 417 475 333 237 514 435 294 371 456 442 264 459 296 507 460 327 257 452 506 533 486 386 498 302 451 412 363 121 538 180 357 457 279 414 432 513 518 519 329 550 541 461 264 289 384 77 490 440 531 564 543 409 440 516 498 175 472 571 79 512 480 403 518 562 563 575 324 499 508 364 257 343 379 489 310 61 278 479 325 526 264 542 543 434 405 312 443 468 594 576 411 490 509 442 579 586 390 568 569 555 591 417 445 564 396 564 476 585 610 604 423 465 608 615 437 475 342 371 555 524 579 533 196 309 616 590 562 408 628 445 283 347 473 631 428 344 421 405 477 600 248 646 415 617 648 452 654 659 277 648 624 598 473 412 556 231 667 619 646 645 554 310 467 338 482 611 598 652 675 594 494 591 509 634 364 634 363 516 589 517 694 498 610 229 242 203 584 683 663 652 654 348 495 643 616 553 657 631 451 422 212 658 692 574 683 675 676 719 669 718 522 601 613 485 371 454 730 727 693 551 371 504 435 456 572 591 734 498 586 713 737 490 736 608 744 728 703 703 637 573 752 754 206 743 687 572 621 612 629 753 495 413 498 652 669 566 459 765 487 612 180 691 773 775 464 713 616 647 351 743 745 488 739 670 776 738 530 748 739 642 767 617 372 439 671 795 782 801 767 468 649 676 795 754 576 659 724 631 791 770 670 710 776 776 780 815 218 683 525 801 741 741 752 796 698 802 714 764 813 810 727 836 772 819 782 474 801 597 505 819 683 764 845 819 709 839 843 838 842 720 849 804 796 716 766 768 812 752 856 852 790 847 804 793 857 867 819 760 776 800 781 716 827 866 646 788 863 618 879 872 617 797 747 750 855 888 872 852 591 875 560 834 810 856 882 702 879 808 830 889 895 866 790 906 902 897 911 885 794 914 901 848 899 914 910 864 805 811 847 917 705 790 862 761 886 912 551 905 913 802 931 879 818 938 914 846 563 927 939 903 828 740 913 914 908 907 932 920 952 902 917 951 902 947 866 929 951 960 921 913 872 930 966 968 937 966 962 971 969 970 969 401 954 489 975 974 943 914 767 984 979 966 966 955 963 979 979 991 950 987 994 974 995 998 1 6 6 8 8 8 19 15 21 23 33 38 22 24 49 53 47 54 58 32 23 30 67 62 57 4 56 51 1 31 14 47 17 81 37 62 38 54 77 64 92 13 73 87 38 97 40 74 90 100 39 66 102 83 81 64 74 89 103 80 41 68 107 79 129 107 29 126 42 13 108 85 52 75 101 128 117 125 60 56 134 115 97 122 110 58 72 51 26 139 94 135 144 96 93 121 138 152 145 70 152 164 113 140 23 67 156 178 161 26 169 168 99 178 145 151 192 126 150 180 176 147 131 54 207 60 160 212 215 209 144 202 216 160 112 26 197 220 13 150 3 41 223 223 193 90 231 102 100 207 99 237 233 167 239 159 179 219 168 239 250 249 79 230 42 272 133 240 104 257 69 231 254 287 173 251 148 264 19 224 171 271 236 264 172 180 186 296 90 64 277 283 233 8 226 261 226 247 236 109 320 231 243 187 127 294 288 265 182 173 185 196 305 76 303 256 308 266 194 329 299 277 296 325 336 331 320 3 253 208 344 185 340 252 227 182 249 191 356 336 153 210 244 285 222 194 256 179 165 355 330 297 367 336 221 170 331 144 220 222 370 51 204 59 393 353 395 313 199 162 345 261 203 385 250 388 319 335 245 136 192 198 326 179 372 328 389 225 387 281 334 368 205 243 398 427 339 321 425 251 421 420 327 280 308 421 222 422 319 358 377 412 357 417 384 344 405 419 449 403 387 410 304 236 441 240 423 392 391 427 294 166 441 364 248 268 398 381 344 446 287 298 283 382 290 466 447 140 410 428 465 453 470 471 429 267 430 433 183 398 257 312 462 275 485 260 364 360 451 463 341 472 443 497 478 181 311 357 438 497 319 399 510 501 295 476 424 450 227 489 318 357 342 189 509 224 350 378 478 477 475 314 346 376 492 435 496 540 332 468 509 500 337 532 515 439 503 359 457 443 506 520 552 534 527 290 502 274 418 406 552 536 455 308 356 558 534 400 459 498 251 420 543 482 365 545 573 574 241 504 443 574 352 510 539 481 416 526 387 302 563 422 544 553 583 491 580 521 551 559 597 570 380 344 484 483 394 188 466 354 286 468 342 599 482 582 588 448 34 492 523 215 575 511 579 435 553 471 481 530 548 528 451 445 330 598 571 601 599 257 431 595 587 517 235 622 571 538 633 472 633 607 579 598 536 577 639 565 638 576 613 446 434 616 339 552 643 537 361 557 393 581 614 607 662 652 626 603 549 558 655 638 263 607 515 556 606 539 645 604 668 446 628 645 583 592 625 671 660 632 546 469 416 636 662 677 669 562 633 641 560 680 666 324 531 558 488 480 620 648 688 636 602 469 674 555 665 696 605 627 678 644 672 569 700 536 297 695 676 712 695 563 379 622 692 681 702 523 725 703 439 682 686 621 679 704 637 704 120 653 634 232 669 717 722 730 632 708 735 545 624 699 567 661 472 684 596 746 535 301 747 697 547 603 711 721 591 742 635 684 664 316 753 529 693 756 766 498 540 692 527 627 569 609 455 717 685 735 240 644 640 668 732 404 690 683 755 630 753 757 665 370 691 727 709 769 779 723 412 735 752 228 768 767 517 714 774 752 674 701 759 739 439 791 734 763 791 783 623 776 693 799 578 769 651 787 531 141 748 783 731 706 802 806 758 797 785 803 718 817 707 733 784 751 777 682 760 827 798 458 593 775 726 836 819 794 729 822 633 596 706 715 835 697 713 840 769 814 786 823 749 809 383 708 844 798 705 818 562 821 361 764 741 324 844 836 736 418 806 807 673 824 668 758 488 668 854 794 573 859 792 851 852 661 858 825 446 869 868 847 849 853 850 710 747 870 810 852 771 766 873 861 874 880 820 857 871 705 650 848 886 876 459 884 875 892 841 885 855 860 832 554 891 594 726 895 860 826 883 898 833 792 831 823 561 868 826 897 778 829 812 874 810 744 907 910 861 862 789 689 881 909 916 758 877 893 922 879 908 575 915 903 924 921 795 887 881 928 656 897 923 886 907 918 907 915 657 933 894 904 939 923 907 865 890 804 872 938 935 905 926 892 837 778 927 946 816 952 939 906 919 940 944 933 951 762 942 800 898 941 949 887 793 951 885 896 958 900 925 797 948 790 946 878 959 839 956 936 952 945 961 953 927 897 964 957 965 966 930 968 967 971 939 963 972 954 976 983 985 942 973 934 967 981 982 980 988 987 966 990 989 987 977 978 985 992 983 987 987 995 986 993 964 974 996 995 989 997 984 999
0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
1033 1465
1421 1908
1022 1853
1270 1470
1192 1353
