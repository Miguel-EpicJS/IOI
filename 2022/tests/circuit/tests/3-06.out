4faa78c22ee35083e49b65601698a1bd98a87a0f
OK
10265418
663812742
838804900
935588604
32034942
